lib/index.coffee

```diff
@@ -0,0 +1,54 @@
+_ = require "underscore"
+async = require "async"
+genericPool = require "generic-pool"
+
+create_pool = (thrift, options) ->
+  genericPool.Pool(
```

Comment:
unnecessary parens for this function call. i see a lot of them in this file that could be removed